static/styles/zulip.css

```diff
@@ -1172,7 +1167,6 @@ div.message_table {
 .message_row {
     position: relative;
     border-left: 1px solid #e2e2e2;
```

Comment:
Yes, we do. Otherwise, the pointer/stream color frame gets misaligned:

![image](https://cloud.githubusercontent.com/assets/7356565/24676986/73b505fe-1985-11e7-8a43-f0d80d324745.png)